uast/uast.go

```diff
@@ -40,6 +40,13 @@ const (
 	BinaryExpressionRight
 	BinaryExpressionOp
 
+	// Infix should mark the nodes with are parents of expression nodes using infix notation, e.g.: a+b.
+	Infix
+	// Prefix should mark the nodes with are parents of expression nodes using prefix notation, e.g.: +ab.
+	Prefix
```

Comment:
I don't think we need the Prefix role at the moment, since that's default, so adding the role is just redundant.